portability-types-client/src/test/java/org/dataportabilityproject/types/client/transfer/DataTransferTest.java

```diff
@@ -13,11 +13,12 @@
     public void verifySerializeDeserialize() throws Exception {
         ObjectMapper objectMapper = new ObjectMapper();
 
-        DataTransfer transfer = new DataTransfer("testSource", "testDestination", "application/json", DataTransfer.Status.INPROCESS);
+        DataTransfer transfer = new DataTransfer("testSource", "testDestination", "application/json", DataTransfer.Status.INPROCESS,"nexturltofollow");
```

Comment:
space after INPROCESS,

Reply:
Done - reformatted the file with intellij autoformatter.